src/consensus/aft/raft.h

```diff
@@ -2898,9 +2940,18 @@ namespace aft
       {
         if (
           nodes.find(id) != nodes.end() && learners.find(id) == learners.end())
+        {
+          LOG_TRACE_FMT("trusted: {}", id);
```

Comment:
```suggestion
          LOG_TRACE_FMT("Trusted: {}", id);
```

Reply:
Unnecessary debug output, I'll remove that.